### PR TITLE
With the deb package the config it uses is localConfig.js and not config.js

### PR DESCRIPTION
**Pull request by klobucar**
@krobertson 
FYI PR

### CODE REVIEW

**Comment by austinmills** [comment]
I believe you'll also want to change files/default/tests/minitest/default_test.rb

File: recipes/default.rb [hunk 1]
```diff
@@ -67,6 +67,11 @@
   only_if { Dir.exists?("/etc/sv/statsd") }
 end
 
+excecute "remove old statsd config" do
+  command "rm #{node['statsd']['conf_dir']/config.js"
+  only_if {File.exists?("#{node['statsd']['conf_dir']}/config.js") }
+end
```
Comment (krobertson) [inline recipes/default.rb]: Cleaner:

```
file "#{node['statsd']['conf_dir']/config.js" do
  action :delete
end
```

**Comment by krobertson** [comment]
+1 to @austinmills 

Would recommend the file resource over an execute resource.

**Comment by klobucar** [comment]
Done and merging.

**Comment by krobertson** [comment]
Line 70 has broken syntax. The #{ is missing the }

**Pull request merged**